### ISSUE TITLE
Can't access a component (e.g. Span) inside a MenuItem of a MenuBar

### ISSUE THREAD

**Issue by hfazai** [mvysny/karibu-testing#76]
This library is really powerful and amazing. Thanks for such great work! :+1: 

I'm adding a span (that contains an icon and a text) to a menu item. I'm unable to access it in my test. This is a code sample:
````KOTLIN
val menuBar = MenuBar()
val project = menuBar.addItem(Span("Project"))
val projectSubMenu = project.subMenu
val users: MenuItem = projectSubMenu.addItem("Users")
add(menuBar)
````

Test sample:
````KOTLIN
val modulesMenu = modulesLayout._get<MenuBar>()
modulesMenu._expect<Span>()
`````
This is the tree shown in the error:
````
/: No visible Span in MenuBar[] matching Span: []. Component tree:
└── MenuBar[]
    └── MenuBarRootItem[]
        └── MenuItem[text='Users']
````

I dont know if this is a bug or I'm missing something in my code?

By the way, the component tree is very helpful. I really appreciate the idea :+1: :100: .

**Comment by mvysny** [comment]
Hi, thank you :) And thanks for letting me know. Looks like Karibu-Testing for some reason can't list children of a MenuItem. Let me investigate :+1:

**Comment by mvysny** [comment]
Fixed in Karibu-Testing 1.3.1

**Comment by hfazai** [comment]
> Fixed in Karibu-Testing 1.3.1

Thanks :)
